src/runtime/ndarray.cc

```diff
@@ -190,6 +193,49 @@ NDArray NDArray::Empty(std::vector<int64_t> shape, DLDataType dtype, DLContext c
   return ret;
 }
 
+NDArray NDArray::NonEmpty(std::vector<int64_t> shape, DLDataType dtype, DLContext ctx) {
```

Comment:
NonEmpty is not a good name, given that what we are really doing is just randomly fill in ndarray, we can instead call it (use a runtime PackedFunc) `.random_fill(arr)`, consider make it part of src/runtime/rpc because it is mainly used in rpc server.

Reply:
consider put it under https://github.com/apache/incubator-tvm/blob/master/src/runtime/contrib/random/random.cc and always switch on random if RPC is enabled